doc/monitoring_containers/monitoring_containers.md

```diff
@@ -0,0 +1,276 @@
+# Monitoring and Auto-Mitigating the Unhealthy of Docker Containers in SONiC
```

Comment:
Reword: `# Monitoring and Auto-Mitigating Unhealthy Containers in SONiC`

Reply:
Good suggestion! Reworded.